PromiseFileReader.js

```diff
@@ -1,6 +1,6 @@
 function readAsDataURL(file) {
-  if (!(file instanceof File)) {
-    throw new TypeError('Must be a File')
+  if (!(file instanceof File || file instanceof Blob)) {

```

Comment:
`file instanceof File` is no-longer needed.

A File inherits Blob. The second statement will always be true if the first statement is true.

https://developer.mozilla.org/en-US/docs/Web/API/File